draft-ietf-quic-transport.md

```diff
@@ -1453,6 +1453,10 @@ return.  A PING frame containing 12 randomly generated {{?RFC4086}} octets is
 sufficient to ensure that it is easier to receive the packet than it is to guess
 the value correctly.
 
+Note:
+
+: Retransmissions of the PING frame MUST also use the same remote address.
+
 If validation of the new remote address fails, after allowing enough time for
```

Comment:
I assume that you refer to the text about checking the old address as well?  I don't see the conflict.  This is talking about retransmissions.

The point is that the PING frame cannot be sent to multiple addresses or it only validates one of those addresses.